### PR TITLE
add toggle to see plain password when adding & editing ciphers (#292)

### PR DESCRIPTION
**Pull request by MoonShade**

### CODE REVIEW

**Comment by MoonShade** [comment]
This is my first contribution on GitHub, I hope this can be helpful and if you can provide me with any feedback to this, or how I can improve, please do!

**Comment by kspearrin** [comment]
Hi. thanks for the PR. Can you include a screenshot of what this looks like?

**Comment by MoonShade** [comment]
It looks like this for editing and creating new ciphers:

bitwarden-passw-visibility

File: src/popup/app/vault/views/vaultEditCipher.html [hunk 1]
```diff
@@ -31,9 +31,17 @@
                             <label for="loginUsername" class="item-label">{{i18n.username}}</label>
                             <input id="loginUsername" type="text" name="Login.Username" ng-model="cipher.login.username">
                         </div>
-                        <div class="list-section-item">
-                            <label for="loginPassword" class="item-label">{{i18n.password}}</label>
-                            <input id="loginPassword" type="password" name="Login.Password" ng-model="cipher.login.password">
+                        <div class="list-section-item flex">
+                            <div class="flex-grow">
+                                <label for="loginPassword" class="item-label">{{i18n.password}}</label>
+                                <input id="loginPasswordPlain" type="text" name="Login.Password.Plain" ng-model="cipher.login.password" ng-if="showPassword">
```
Comment (kspearrin) [inline src/popup/app/vault/views/vaultEditCipher.html]: You should be able to do this with 1 input and just change the `type` attribute with angular directives.

Comment (MoonShade) [reply src/popup/app/vault/views/vaultEditCipher.html]: I'm now using 1 input and added a conditional value for the type attribute, I think that should solve this problem.

**Pull request merged**